google-map.html

```diff
@@ -511,217 +147,295 @@
 
-    /**
-     * A comma separated list (e.g. "places,geometry") of libraries to load
-     * with this map. Defaults to "places". For more information see
-     * https://developers.google.com/maps/documentation/javascript/libraries.
-     *
-     * @attribute libraries
-     * @type string
-     * @default "places"
-     */
-    libraries: "places",
+      /**
+       * A latitude to center the map on.
+       *
+       * @attribute latitude
+       * @type number
+       * @default 37.77493
+       */
+      latitude: {
+        type: Number,
+        value: 37.77493
+      },
 
-    /**
-     * A longitude to center the map on.
-     *
-     * @attribute longitude
-     * @type number
-     * @default -122.41942
-     */
-    longitude: -122.41942,
+      /**
+       * A comma separated list (e.g. "places,geometry") of libraries to load
+       * with this map. Defaults to "places". For more information see
+       * https://developers.google.com/maps/documentation/javascript/libraries.
+       *
+       * @attribute libraries
+       * @type string
+       * @default "places"
+       */
+      libraries: {
+        type: String,
+        value: "places"
+      },
 
-    /**
-     * A zoom level to set the map to.
-     *
-     * @attribute zoom
-     * @type number
-     * @default 10
-     */
-    zoom: 10,
+      /**
+       * A longitude to center the map on.
+       *
+       * @attribute longitude
+       * @type number
+       * @default -122.41942
+       */
+      longitude: {
+        type: Number,
+        value: -122.41942
+      },
 
-    /**
-     * When set, prevents the map from tilting (when the zoom level and viewport supports it).
-     *
-     * @attribute noAutoTilt
-     * @type boolean
-     * @default false
-     */
-    noAutoTilt: false,
+      /**
+       * A zoom level to set the map to.
+       *
+       * @attribute zoom
+       * @type number
+       * @default 10
+       */
+      zoom: {
+        type: Number,
+        value: 10,
+        observer: 'zoomChanged'
+      },
 
-    /**
-     * When set, displays a map marker at the center point.
-     *
-     * @attribute showCenterMarker
-     * @type boolean
-     * @default false
-     */
-    showCenterMarker: false,
+      /**
+       * When set, prevents the map from tilting (when the zoom level and viewport supports it).
+       *
+       * @attribute noAutoTilt
+       * @type boolean
+       * @default false
+       */
+      noAutoTilt: {
+        type: Boolean,
+        value: false
+      },
 
-    /**
-     * Map type to display. One of 'roadmap', 'satellite', 'hybrid', 'terrain'.
-     *
-     * @attribute mapType
-     * @type string
-     * @default roadmap
-     */
-    mapType: 'roadmap', // roadmap, satellite, hybrid, terrain
+      /**
+       * When set, displays a map marker at the center point.
+       *
+       * @attribute showCenterMarker
+       * @type boolean
+       * @default false
+       */
+      showCenterMarker: {
```

Comment:
I'd actually like to deprecate this now that we have `<google-map-marker>`: https://github.com/GoogleWebComponents/google-map/issues/109